constants/constants.F90

```diff
@@ -37,12 +37,24 @@ module constants_mod
 
 real :: realnumber !< dummy variable to use in HUGE initializations
 
+#ifdef SMALL_EARTH
+#if defined(DCMIP) || (defined(HIWPP) && defined(SUPER_K))
+ real, public, parameter :: small_fac =  1._r8_kind / 120._r8_kind #only needed for supercell test
+#elif defined(HIWPP)
+ real, public, parameter :: small_fac = 1._r8_kind / 166.7_r8_kind
+#else
+ real, public, parameter :: small_fac = 1._r8_kind / 10._r8_kind
+#endif
+#else
+ real, public, parameter :: small_fac = 1._r8_kind
+#endif
+
 #ifdef GFS_PHYS
 ! SJL: the following are from fv3_gfsphysics/gfs_physics/physics/physcons.f90
-real,               public, parameter :: RADIUS = 6.3712e+6_r8_kind           !< Radius of the Earth [m]
-real(kind=r8_kind), public, parameter :: PI_8   = 3.1415926535897931_r8_kind  !< Ratio of circle circumference to diameter [N/A]
-real,               public, parameter :: PI     = 3.1415926535897931_r8_kind  !< Ratio of circle circumference to diameter [N/A] (REAL(KIND=8))
-real,               public, parameter :: OMEGA  = 7.2921e-5_r8_kind   !< Rotation rate of the Earth [1/s]
+real,               public, parameter :: RADIUS = 6.3712e+6_r8_kind * small_fac !< Radius of the Earth [m]
```

Comment:
Is this answer changing?

Reply:
As this is gated by a #ifdef macro for GFS_PHYS, any answer changes will be limited to SHiELD and UFS models and is already in use by SHiELD. The GFDL models will be unaffected.